LoopKitUI/Extensions/UIColor+LoopKit.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  UIColor.swift
+//  LoopKitUI
+//
+//  Created by Anna Quinlan on 7/1/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+import UIKit
+
+extension UIColor {
+    @nonobjc static let COBTintColor: UIColor = {
+        if #available(iOSApplicationExtension 13.0, iOS 13.0, *) {
```

Comment:
I know this was copy/pasted, but since our deployment target has updated to 13.0 since this code was written, we might as well drop the availability check here.